dotnet/src/Skills/Skills.UnitTests/OpenAPI/OpenApiDocumentParserV31Tests.cs

```diff
@@ -92,7 +92,7 @@ public async Task ItCanParsePutOperationMetadataSuccessfullyAsync()
         var putOperation = operations.Single(o => o.Id == "SetSecret");
         Assert.NotNull(putOperation);
         Assert.Equal("Sets a secret in a specified key vault.", putOperation.Description);
```

Comment:
nit: tests for the optional parameter in `GetParameters` would be good, feel free to resolve for follow up though.